lib/sqlalchemy/ext/asyncio/session.py

```diff
@@ -864,7 +864,11 @@ def get_bind(self, mapper=None, clause=None, **kw):
             mapper=mapper, clause=clause, bind=bind, **kw
         )
 
-    async def connection(self, **kw: Any) -> AsyncConnection:
+    async def connection(
+        self,
+        bind_arguments: Optional[_BindArguments] = None,
+        execution_options: Optional[_ExecuteOptions] = None,
+    ) -> AsyncConnection:
```

Comment:
Not sure about this one. I assume there are tests for these parameters in place somewhere but I couldn't find them.

In any case, since these are the only two arguments the underlying session accepts it seemed logical to add them here.

Reply:
The kw args should be kept, since they are supported in the session.connection method

Reply:
You're right, that's an oversight on my end. I'll add them back